### PR TITLE
CI + CD for browser builds

### PR DESCRIPTION
**Pull request by thomasballinger**
CI + (CD on pushes to master)

This requires a repo environment called "aws-deploy," which you can create empty to get the CI working.

### CODE REVIEW

**Comment by quyykk** [comment]
> This requires a repo environment called "aws-deploy," which you can create empty to get the CI working.

~~What do you mean by that?~~ nvm figured it out. thanks!

File: .github/workflows/browser.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: browser
+
+on:
+  push:
```
Comment (quyykk) [inline .github/workflows/browser.yml:4]: Do we need to add something here?

Comment (thomasballinger) [reply .github/workflows/browser.yml:4]: [This syntax seems right](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration) so syntax-wise I don't think so, but yes if the desired behavior is not to run on push to any branch.

> You must append a colon (:) to all events, including events without configuration.

Comment (quyykk) [reply .github/workflows/browser.yml:4]: > desired behavior is not to run on push

Now I'm confused. Why not?

Comment (thomasballinger) [reply .github/workflows/browser.yml:4]: Sorry, that was confusing. The clearer answer is "No, we do not need to add something here."

File: .github/workflows/browser.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Build
+        run: source emsdk/emsdk_env.sh; make output/index.html 
+      - name: Deploy
+        if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push' }}
+        run: source emsdk/emsdk_env.sh; make deploy
```
Comment (quyykk) [inline .github/workflows/browser.yml]: We can build in parallel by passing `-j $(nproc)` to make (to both calls).

**Review by quyykk — APPROVED** [review]
LGTM

**Pull request merged**